### ISSUE TITLE
Question about Ridge plots

### ISSUE THREAD

**Issue by aditisk** [enblacar/SCpubr#8]
Hi @enblacar,

Is it possible to make a ridge plot showing the expression of a certain gene ? If I change the features to a gene name I get the following error:

`Error: 'SCT' is not an assay`

As per your suggestion in my previous issue, I am explicitly specifying `assay="RNA"`. The code from the vignette plotting `nFeature_RNA` works but once I change it to a gene name I am getting this error.

**Comment by enblacar** [comment]
Hi @aditisk,

By default, `assay` parameter is set to `SCT`, so if you are working with an `RNA` assay, you will have to use `assay = "RNA"` every time you use the function. 

Then, using it alongside the feature you want to plot should work. If not, please send me the code you are using.

Best.
Enrique

**Comment by aditisk** [comment]
Hi @enblacar, I was already specifying `assay="RNA` but it doesn't seem to work for me. Please see the code below:

`SCpubr::do_RidgePlot(sample =sample,feature = "LAG3",assay = "RNA")
Error: 'SCT' is not an assay`

**Comment by enblacar** [comment]
Hi @aditisk,

I found the source of the error. It is an internal bug in the code in which `SCT` assay was hardcoded by mistake. Is it fixed in the last commit and will also be fixed in the next CRAN update. In the meantime, I offer you the following solution:

```r
devtools::install_github("enblacar/SCpubr", ref = "088f20b2027bacefcf6e38d50e3b373e734ec094")
```

As always, please have in mind that development versions might also contain other bugs, so once you are done with this specific plotting, revert back to the CRAN version of `SCpubr`.

Thanks for this valuable feedback and hope this helps!
Enrique

**Comment by aditisk** [comment]
Hi @enblacar,  I used the version you mentioned above and it works. Thanks :)

**Comment by enblacar** [comment]
All the changes have been implemented in SCpubr v1.0.3, currently available on CRAN.

Best,
Enrique
